app/helpers/answer-options.js

```diff
@@ -1,11 +1,11 @@
-const { formatAnswerArray } = require('./../helpers/standardised-grant-amounts-array')
+const { formatAnswerArray } = require('./../helpers/reference-grant-amounts-array')
```

Comment:
you've updated this import but not the file name... -_-

Reply:
good spot thanks